### ISSUE TITLE
can i use this module without controller 

### ISSUE THREAD

**Issue by prasadjain05** [gianluca-nitti/CH375-Arduino#2]
can i use CH375B USB Bus General Interface Module without controller i tried but does not give data from usb drive...????

**Comment by gianluca-nitti** [comment]
What do you mean by "controller"?
The CH375 can be interfaced with any device with an UART serial interface or by implementing (for example bit-banging) the 8-bit parallel protocol described in the [datasheet](ftp://imall.iteadstudio.com/Modules/IM131009001/DS_IM131009001.pdf). This repository contains a (very basic) library to interface it to an Arduino-compatible device (I used it on ESP8266 with [ESP8266 Arduino](https://github.com/esp8266/Arduino)) through the UART.
What device are you trying to interface it to?

**Comment by prasadjain05** [comment]
i have stm32 device which is i have to configure with ch375B module but i am not getting proper sequence of command which should i fire 1st and secound..so on so if any tutorial or document which can help  me out except datasheet

**Comment by gianluca-nitti** [comment]
Unfortunately I never used STM32 and there isn't much documentation about CH375.
If what you're trying to do is file transfer [this tutorial](https://arduinobasics.blogspot.com/2015/05/ch376s-usb-readwrite-module.html) might be useful, note however that it's not for CH375 but for the CH376S (very similar IC from the same company, [datasheet here](https://www.mpja.com/download/ch376ds1.pdf); most commands are the same).
Also, there are code examples in ZIP files that can be downloaded from [this page](http://www.wch.cn/product/CH375.html) (I used Google Translate for the webpage and the comments in the source files in the zips). IIRC they are for the Intel MCS-51 MCU and use the 8-bit parallel protocol instead of the serial UART, but they gave me a decent understanding of the proper command sequences when I worked on the library in this repo.
